content/source/docs/enterprise-beta/sentinel/manage-policies.html.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: enterprise2
+page_title: "Sentinel - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-sentinel-manage-policies"
+---
+
+# Managing Policies
+
+Sentinel Policies are rules which are enforced on every workspace run to validate the terraform plan and corresponding resources are in compliance with company policies. Policies are made up of a sentinel policy file, a name, and an enforcement level. They are managed at an organization level and a user must be in the organization's owners team to create, edit or delete policies.
+
+To view the policies navigate to the organization's settings page under "Sentinel Policy". From the "Sentinel Policy" section you can view, list, edit, create and delete the policies.
+
+To create a new policy navigate to “Create Policy”. Sentinel is designed to enable policy as code. As such, it is not recommended to use the Terraform Enterprise UI to manage policy. Instead, it is [recommended to store Sentinel policies in version control](./integrate-vcs.html).
```

Comment:
Maybe note that it is useful for simple policies or demos, but complex integrations should be done via  VCS/CI